consensus/types/src/lib.rs

```diff
@@ -97,7 +97,7 @@ pub mod slot_data;
 #[cfg(feature = "sqlite")]
 pub mod sqlite;
 
-use ethereum_types::{H160, H256};
+pub use ethereum_types::{H160, H256};
```

Comment:
I don't think we want these names public, as usually `H160` is aliased as `Address`, and `H256` gets called `Hash256`.

```suggestion
use ethereum_types::{H160, H256};
```